diff --git a/hyworld2/worldgen/video_gen.py b/hyworld2/worldgen/video_gen.py
index 6b42100..cb7f519 100755
--- a/hyworld2/worldgen/video_gen.py
+++ b/hyworld2/worldgen/video_gen.py
@@ -51,7 +51,7 @@ if __name__ == '__main__':
     device = torch.device(f"cuda:{local_rank}")
     torch.cuda.set_device(local_rank)
     dist.init_process_group(
-        backend="cpu:gloo,cuda:nccl",
+        backend="gloo",
         rank=rank,
         world_size=world_size,
     )
