diff --git a/hyworld2/panogen/hunyuan_image_3/modeling_hunyuan_image_3.py b/hyworld2/panogen/hunyuan_image_3/modeling_hunyuan_image_3.py
index e7664c9..ec32815 100755
--- a/hyworld2/panogen/hunyuan_image_3/modeling_hunyuan_image_3.py
+++ b/hyworld2/panogen/hunyuan_image_3/modeling_hunyuan_image_3.py
@@ -930,7 +930,7 @@ class HunyuanStaticCache(StaticCache):
         """
         cache_position = cache_kwargs.get("cache_position")
         if self.layers[layer_idx].keys is None:
-            self.layers[layer_idx].lazy_initialization(key_states)
+            self.layers[layer_idx].lazy_initialization(key_states, value_states)
         k_out = self.layers[layer_idx].keys
         v_out = self.layers[layer_idx].values
 
